Issue #3: Fix DBI_CryptType crypt

esabol opened the pull request:
Refer to issue #3.

This completely removes the incorrect `_crypt_digest`, but nothing else uses it, I think.

It also fixes a couple of mispellings in the pod text. If you prefer that in a separate PR, I can do that.

Would you like me to update the Changes file as well?

esabol committed:
Issue #3: Fix DBI_CryptType crypt

esabol committed:
Issue 3: Slight tweak to coding style

matisse commented:
I think the test `t/utils.t` needs to be updated also.

You can update the Changes file or i can do it when I eventually submit a new version to CPAN.

esabol committed:
Fix tests for Issue 3 (DBI_CryptType crypt)

esabol committed:
Tweak test for Issue 3 (DBI_CryptType crypt)

esabol commented:
Updated `t/utils.t`.

Also, apparently unrelated, my testing revealed the error `Can't locate object method "quote_identifier" via package "DBI::Mock::dbh"`, so I added a simplistic `quote_identifier` method to `t/mock_libs/DBI.pm`. If you prefer that to be fixed in a separate PR, let me know.

esabol committed:
Minor whitespace tweaks for Issue 3 (DBI_CryptType crypt)

matisse commented:
Thank you Ed. Unless you want to add any more changes I will go ahead and merge this and prepare a new version for CPAN.

esabol commented:
I’m done with changes for this issue. I am hoping to work on adding support for Apache 2.4, but that probably will be several weeks from now, if I had to guess.

matisse commented:
Uploaded to CPAN - thank you very Much!

I mentioned you in the Changes file, I hope that is OK.


-Matisse



> On Aug 17, 2019, at 10:45 AM, Ed Sabol <notifications@github.com> wrote:
> 
> I’m done with changes for this issue. I am hoping to work on adding support for Apache 2.4, but that probably will be several weeks from now, if I had to guess.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/matisse/Apache-AuthCookieDBI/pull/4?email_source=notifications&email_token=AAEBFLLRHLKURJB67GHNAYDQFA2KVA5CNFSM4ILY3WI2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD4QQI5Q#issuecomment-522257526>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAEBFLJZORCOEOCTFKZSLE3QFA2KVANCNFSM4ILY3WIQ>.
>

esabol commented:
That’s great!
